src/main/java/org/sonar/squidbridge/annotations/NoSqale.java

```diff
@@ -24,6 +24,11 @@
 import java.lang.annotation.RetentionPolicy;
 import java.lang.annotation.Target;
 
+/**
+ * @deprecated since 2.7. Will be removed without alternative.
+ * Since SQ 5.6 LTS, SQALE is not used in our model and the NoSqale annotation has no effect.
+ */
+@Deprecated
 @Retention(RetentionPolicy.RUNTIME)
 @Target(ElementType.TYPE)
 public @interface NoSqale {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Do not forget to remove this deprecated code someday. ([squid:S1133](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1133))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=org.sonarsource.sslr-squid-bridge%3Asslr-squid-bridge&branch=SSLRSQBR-71&issues=AWHXxlpMaf12pdu-6jE8&open=AWHXxlpMaf12pdu-6jE8)

[ISSUE_KEY]: AWHXxlpMaf12pdu-6jE8